lib/linguist/languages.yml

```diff
@@ -1036,6 +1036,18 @@ Frege:
   - .fr
   tm_scope: source.haskell
   ace_mode: haskell
+  
+FreeMarker:
+  type: programming
+  color: "#0050b2"
+  group: Java
+  search_term: ftl
```

Comment:
`search_term` is now deprecated and it should be necessary here since `ftl` is already an alias.